common/src/main/webapp/sass/visualization.scss

```diff
@@ -25,12 +25,12 @@
   }
 
   .gmos-probe-arm {
-    stroke: hsl(0, 67.9%, 41.6%);
+    stroke: hsl(0deg 67.9% 41.6%);
```

Comment:
Didn't know this notation

Reply:
When `fix` switched from hex codes, it used this notation.